app/controllers/camaleon_cms/camaleon_controller.rb

```diff
@@ -51,7 +51,7 @@ def render_error(status = 404, exception = nil, message = '')
     # generate captcha image
     def captcha
       image = cama_captcha_build(params[:len])
-      send_data image.to_blob, type: image.mime_type, disposition: 'inline'
+      send_data image.to_blob, type: MiniMime.lookup_by_extension(image.type).content_type, disposition: 'inline'
```

Comment:
This seems good. It could be further simplified if desired, because `cama_captcha_build` [always returns a .jpg](https://github.com/owen2345/camaleon-cms/blob/6074276104252247e0c12f0bb07c25e9fff6991b/app/helpers/camaleon_cms/captcha_helper.rb#L5), so we could just use "image/jpeg" here and skip the method call. This version might be more resilient if that method is ever changed, though.

Reply:
Yeah, let it stay resilient.